modules/exploits/linux/local/docker_cgroup_escape.rb

```diff
@@ -83,13 +83,17 @@ def check
     print_status('Unable to determine host OS, this check method is unlikely to be accurate if the host isn\'t Ubuntu')
     release = kernel_release
     # https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-0492
-    release_short = Rex::Version.new(release.split('-').first)
-    release_long = Rex::Version.new(release.split('-')[0..1].join('-'))
-    if release_short >= Rex::Version.new('5.13.0') && release_long < Rex::Version.new('5.13.0-37.42') || # Ubuntu 21.10
-       release_short >= Rex::Version.new('5.4.0') && release_long < Rex::Version.new('5.4.0-105.119') || # Ubuntu 20.04 LTS
-       release_short >= Rex::Version.new('4.15.0') && release_long < Rex::Version.new('4.15.0-173.182') || # Ubuntu 18.04 LTS
-       release_short >= Rex::Version.new('4.4.0') && release_long < Rex::Version.new('4.4.0-222.255') # Ubuntu 16.04 ESM
-      return CheckCode::Vulnerable("IF host OS is Ubuntu, kernel version #{release} is vulnerable")
+    begin
+      release_short = Rex::Version.new(release.split('-').first)
+      release_long = Rex::Version.new(release.split('-')[0..1].join('-'))
```

Comment:
since we re-join with '-' (which works on ubuntu where I tested it), this will fail on amazon linux.